packages/components/src/autocomplete/index.tsx

```diff
@@ -241,6 +239,7 @@ export function useAutocomplete( {
 		if ( isCollapsed( record ) ) {
 			return getTextContent( slice( record, 0 ) );
 		}
+		return null;
```

Comment:
Should `textContent` be an empty string instead of `null`? I see we're using it as a string so a `null` might cause issues:

https://github.com/WordPress/gutenberg/blob/15b401a4c0e4335e6bb7d6540c2c1b13aa3d77a1/packages/components/src/autocomplete/index.tsx#L253

Reply:
Good point